packages/aws-cdk/bin/cdk.ts

```diff
@@ -66,7 +66,7 @@ async function parseCommandLineArguments() {
       .option('strict', { type: 'boolean', desc: 'do not filter out AWS::CDK::Metadata resources', default: false }))
     .command('metadata [STACK]', 'Returns all metadata associated with this stack')
     .command('init [TEMPLATE]', 'Create a new, empty CDK project from a template. Invoked without TEMPLATE, the app template will be used.', yargs => yargs
-      .option('language', { type: 'string', alias: 'l', desc: 'the language to be used for the new project (default can be configured in ~/.cdk.json)', choices: initTemplateLanuages })
+      .option('language', { type: 'string', alias: 'l', desc: 'the language to be used for the new project (default can be configured in ($CDK_HOME|~)/.cdk.json)', choices: initTemplateLanuages })
```

Comment:
just say $CDK_HOME and at the end say it defaults to `~`. No need to indicate the default every time.